handlers/auth.go

```diff
@@ -24,3 +24,19 @@ func Auth(secret string) gin.HandlerFunc {
 		c.Set("user", user)
 	}
 }
+
+func getUserFromJwt(webtoken, secret string) (*models.User, error) {
+	token, err := jwt.Parse(webtoken, func(token *jwt.Token)(interface{}, error){
```

Comment:
webtoken is the token string that's returned by /sessions/new. Not sure if we can combine them, unless we want to manually want to get the Authorization header from the request. The Auth middleware uses jwt.ParseFromRequest, while getUserFromJwt receives the token as a string.